packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -302,6 +309,12 @@ abstract contract Staker is StakerInterface, Ownable, Lockable {
         return block.timestamp;
     }
 
+    function isVoterActivelySignaledOnEmergencyAction(address voterAddress) public view returns (bool) {
```

Comment:
see here for the intra-calls to block if you are currently signed on a safety action.